stories/utils/StyleApi/StyleApi.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
```

Comment:
This file is handling two concerns that needs to be split.

1. scrape the data
1. render the settings panel

the first one, is something that should be done during "built time", or more accurate during the webpack bundling of the storybook.

The second one is already separated so it's good. this file should just aggregate the parsed data and render the table and as a next step - an automated panel